features.yaml

```diff
@@ -208,12 +208,20 @@ features:
     link: "/docs/concepts/security/#authorization"
     level:
       checklist: features/authorization.md
-      maturity: Beta
+      maturity: Stable
       nextExpectedPromotion: ""
     area: Security and policy enforcement
   - name: "End User (JWT) Authentication"
     id: "security.jwt_authentication"
-    link: "/docs/concepts/security/#authentication"
+    link: "/docs/reference/config/security/request_authentication/"
+    level:
+      checklist: ""
+      maturity: Stable
+      nextExpectedPromotion: ""
```

Comment:
If we are promoting this to stable, we should really have a filled out checklist. It looks like we could start with what's in this PR: https://github.com/istio/enhancements/pull/42.

Reply:
somehow there is no `jwt_authentication.md` in `/features`, should I create a new one for jwt authentication promotion ?

Reply:
jwt_authentication.md is a new file in https://github.com/istio/enhancements/pull/42, which has not been merged. I think you can create a jwt_authentication.md (or an equivalent) as the checklist of RequestAuthentication.

Reply:
Meanwhile, you may want to refer to the checklist of another stable feature when filling the checklist of RequestAuthentication.